safety/errors.py

```diff
@@ -12,3 +12,16 @@ class InvalidKeyError(DatabaseFetchError):
 
 class TooManyRequestsError(DatabaseFetchError):
     pass
+
+
+class NetworkConnectionError(DatabaseFetchError):
+    pass
+
+
+class RequestTimeoutError(DatabaseFetchError):
+    pass
+
+
+class ServerError(DatabaseFetchError):
+    pass
```

Comment:
As you catch all of them and just change the message, not to mention they have all the same class, give a custom message to these classes so you don't need to repeat when catching them.